logger/sqlite.go

```diff
@@ -70,6 +70,14 @@ func dumpDatabase(dbFilePath, dumpFilePath string) error {
 }
 
 func rebuildDatabase(dumpFilePath, newDbFilePath string) error {
+	// Remove the new database file if it already exists
+	if _, err := os.Stat(newDbFilePath); err == nil {
+		err = os.Remove(newDbFilePath)
+		if err != nil {
+			return fmt.Errorf("failed to remove existing new database file: %v", err)
+		}
+	}
+
```

Comment:
@mazar-farran looks good to me, just a slight addition for this logic:
```err = rebuildDatabase(dumpFile, newDB)
	if err != nil {
		log.Fatalf("Failed to rebuild database: %v", err)
	}
	```
Maybe there's no need to do a fatal log that will terminate the service if there were any issues rebuilding DB?
Potentially we can just delete DB file if there're any issues repairing it, and that's what we can do and start new DB file instead

Reply:
agreed